test/manipulation.test.js

```diff
@@ -176,18 +176,6 @@ describe('manipulation', function() {
         .catch(done);
     });
 
-    it('should work when called without callback', function(done) {
-      Person.afterCreate = function(next) {
```

Comment:
If we want to be correct, then this should have to be rewritten using `after save` hook. However, since this test was written, the API was slightly changed and a callback-less invocation returns a promise, which is already tested elsewhere, and thus it should be safe to remove this test (and other tests similar to this one).